### PR TITLE
Am pm

### PR DESCRIPTION
**Pull request by znotdead**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -406,6 +410,10 @@ <h3>License</h3>
 var input = $('#single-input').clockpicker({
 	placement: 'bottom',
 	align: 'left',
+  twelvehour: true,
```
Comment (Rj7) [inline index.html:413]: formatting.

File: src/clockpicker.js [hunk 2]
```diff
@@ -5,725 +5,767 @@
...
+    this.bg.setAttribute('cy', cy);
+    this.fg.setAttribute('cx', cx);
+    this.fg.setAttribute('cy', cy);
+  };
+
+  // Hours and minutes are selected
+  ClockPicker.prototype.done = function() {
+    raiseCallback(this.options.beforeDone);
+    this.hide();
+    var last = this.input.prop('value'), value;
```
Comment (Rj7) [inline src/clockpicker.js:715]:
```suggestion
    var last = this.input.prop('value');
```
is that a typo?

**Review by Rj7 — CHANGES_REQUESTED** [review]

File: src/clockpicker.js (hunk 2 above)
Comment (znotdead) [reply src/clockpicker.js:715]: it's not a typo. it's short for:
```
var last = this.....;
var value;
```

**Review by Rj7 — APPROVED** [review]

**Review by Rj7 — APPROVED** [review]

File: src/clockpicker.js (hunk 2 above)
Comment (Rj7) [reply src/clockpicker.js:715]: just making sure that what you meant.

**Comment by Rj7** [comment]
@znotdead I need write access to merge.

**Comment by znotdead** [comment]
> @znotdead I need write access to merge.

Done

**Pull request merged**